scripts/installer/windows/product.wxs

```diff
@@ -85,6 +85,16 @@
           <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
         </InstallUISequence>
 
+        <CustomAction Id="InstallPromptFlowService"
```

Comment:
will the service uninstall if we uninstall the program?
what's the behavior in this case?

Reply:
Add command to delete service when uninstall